src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -34,18 +46,38 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
         onDeleteConfirm();
     };
 
+    const handleMenuClose = () => {
+        setIsMenuOpen(false);
+    };
+
+    const handleMenuOpen = () => {
+        setIsMenuOpen(true);
+    };
+
     const tetherProps = {
         attachment: 'top right',
         className: 'bcs-AnnotationActivity-deleteConfirmationModal',
         constraints: [{ to: 'scrollParent', attachment: 'together' }],
         targetAttachment: 'bottom right',
     };
 
+    React.useEffect(() => {
+        if (isConfirmingDelete || isMenuOpen) {
+            onMenuOpen();
```

Comment:
Can we pass these props straight through to `dropdownProps` rather than using an effect?

Reply:
The end goal is to maintain a `isMenuOpen` state that consists of the `DropdownMenu` and `DeleteConfirmation` states. Passing the props straight to `dropdownProps` wouldn't be sufficient alone